knockout-pre-rendered.js

```diff
@@ -451,21 +549,38 @@
           fieldValue = value['convert'](fieldValue);
       }
 
-      // Find the field accessor. If the init binding does not point to an observable
-      // or the field parameter doesn't, we try the text and value binding
-      var fieldAccessor = (ko.isObservable(value) ? value : undefined) || 
-                          (isPlainObject(value) ? value['field'] : undefined) ||                             
-                           allBindings.get('text')      ||
-                           allBindings.get('textInput') ||
-                           allBindings.get('value')     ||
-                           allBindings.get('checked')   ||
-                           allBindings.get('html')      ||
-                           allBindings.get('visible')   ||
-                           allBindings.get('enable')    ||
-                           allBindings.get('disable');
-
-      // Finally, update the observable with the value
+    // Look for property writers for explicit fields in the init binding.
+    var initPropertyWriters = allBindings.get('_ko_prerender_initPropertyWriters') || {};
```

Comment:
This also looks like a private field. See my previous question.

Reply:
Custom name internal to **knockout-pre-rendered**.